### DISCUSSION TITLE
Screen output inconsistency when using list model with delegate SetSpacing=0 and filters

### DISCUSSION THREAD

**Discussion by fabio42** [charmbracelet/bubbletea#629] [Q&A] [answered]
Hello,

I'm playing with the list model and its `list.DefaultDelegate`.
When setting spacing to 0 `SetSpacing(0)` and there is enough items on screen to have multiple pages, I'm observing some strange behaviour when switching between the default view and the filter view. In this setup the Filter line is temporary moving one line up and then list screen is a bit messy, see the attached gif as it is hard to describe.

The following diff is enough to trigger this for me:

```diff
--- a/examples/list-fancy/delegate.go
+++ b/examples/list-fancy/delegate.go
@@ -9,6 +9,7 @@ import (
 func newItemDelegate(keys *delegateKeyMap) list.DefaultDelegate {
 	d := list.NewDefaultDelegate()
 
+	d.SetSpacing(0)
 	d.UpdateFunc = func(msg tea.Msg, m *list.Model) tea.Cmd {
 		var title string
```

Am I missing anything or doing anything wrong here ? Or is this a small bug ?
I just started playing with your libs, and I love everything I'm seeing there! Thank you :)  

![bubbletea_odd](https://user-images.githubusercontent.com/3433725/209584629-d3717ac9-483c-4d31-a9fe-02e49e4726c7.gif)

**Comment by bashbunni** [comment]
Hey @fabio42 what terminal emulator are you using? I wasn't able to reproduce this with Kitty, but am interested in seeing if this is a reproducible bug

**Comment by fabio42** [comment, marked as answer]
Hi @bashbunni, thank you for your feedback, I'm using `Alacritty` but I'm always into a `tmux` session.
And it seems to be cause of the issue (`tmux`), I tried without it, and indeed no issue, I installed kitty, and I trigger this as soon as I run the program inside `tmux`.

**Reply by bashbunni** [reply]
Awesome, thanks for letting us know!
